### ISSUE TITLE
Some examples seem to be broken in the Data Files section

### ISSUE THREAD

**Issue by zhangxiubo** [Nuitka/Nuitka-website#73]
Under the [Data Files](https://nuitka.net/user-documentation/user-manual.html#package-data-include-package-data-package) section, there are a couple of examples showing how to include data files for a package. There is a paragraph stating the following:

> You can use patterns for the filenames as indicated. After a `:` optionally, a filename pattern may be specified as well, selecting only matching files.

However, the examples provided below it used `=` to separate the patterns.  I tried the option myself, and it seems that using `=` as the delimiter causes Nuitka to complain about it --- could this be a typo?

**Comment by kayhayen** [comment]
Good catch, I think the help file texts of Nuitka are itself giving those wrong examples, and they need to be corrected in Nuitka command line help as well.

**Comment by kayhayen** [comment]
I pushed the change, CI will pick it up, thanks.

**Comment by kayhayen** [comment]
This has been online and changed in Nuitka for a while both, thanks for your report.
